### DISCUSSION TITLE
「元宇宙、AI、半導體」夯 私大科系改名拚招生

### DISCUSSION THREAD

**Discussion by Xuewen628** [1101-GS3073-C/Python-Community-GS3073-C#584] [General]
https://tw.news.yahoo.com/%E5%85%83%E5%AE%87%E5%AE%99-ai-%E5%8D%8A%E5%B0%8E%E9%AB%94-%E5%A4%AF-%E7%A7%81%E5%A4%A7%E7%A7%91%E7%B3%BB%E6%94%B9%E5%90%8D%E6%8B%9A%E6%8B%9B%E7%94%9F-054331996.html
最近AI等等的科技發展日益盛大，許多的學校也順勢將之加入教學內容來跟上科技發展及吸引學生

**Comment by Raywei0214** [comment]
去了還是學店，雖說會去的頭腦本來就不太靈光就是了

**Comment by manipulateed** [comment]
看來這改名措施與配套措施完全沒有深思熟慮，資訊的公佈也沒有讓考生更明白，招生的條件也沒有根據，讓學生無從準備，只能說能往前面的學校走就走大步點，學校自然會配合科技趨勢開出更符合需求的通識課。

**Comment by awkward-willy** [comment]
現在各大產業都講求資訊科技的結合與應用，也因此出現許許多多相關的課程、補習班等等，現在連學校都跟進了，看來資訊科技在生活中被廣泛應用是必然的結果，期望未來各科系的大家能靈活運用，持續改善大家的生活。

**Comment by larry40305** [comment]
因為少子化，現在這種比較收不到人的學校必須要想出更多方法來增加招生誘因。

**Comment by magicmicheal** [comment]
雖然這些看準二類大學科系熱門關鍵字，像是半導體大數據，元宇宙這些新領域，但感覺就只是換了菜名，整個口味都沒變，這樣真的是好的嗎？

**Comment by 0212Jerry** [comment]
元宇宙固然熱門，但實際做起來還是有很多障礙需要突破，何況半導體和Ai 也是如此。而新的產業會不斷的出現，我們要做的就是讓自己成為各方面都難以取代都人才。

**Comment by Vincent20030314** [comment]
雖然看似熱門，但是做起來的難度不低，感覺效益不高。
